File: Codigo fuente/Blog.Domain/Entities/Comment.cs
namespace Blog.Domain.Entities;

public class Comment
{
    public Guid Id { get; set; }
    public User Owner { get; set; }
    public Article Article { get; set; }
    public string Body { get; set; }
    public DateTime DatePublished { get; set; }
    public string? Reply { get; set; }
    public bool IsPublic { get; set; }
    public bool IsApproved { get; set; }
    public bool IsEdited { get; set; }
    public IEnumerable<OffensiveWord> OffensiveContent { get; set; }
    
    
}

File: Aplicacion/Blog-Frontend/src/_type/comment.ts
import { OffensiveWord } from './offensiveWord';

export interface Comment {
  id: string;
  articleId: string;
  ownerUsername: string | undefined;
  body: string;
  reply?: string;
  isPublic: boolean;
  isApproved: boolean;
  isEdited: boolean;
  datePublished: number;
  offensiveContent: OffensiveWord[];
  isOffensive: boolean;
}
